src/airtable.js

```diff
@@ -1,16 +1,75 @@
-const Airtable = require("airtable");
+const Airtable = require('airtable');
 
 const base = new Airtable({ apiKey: process.env.AIRTABLE_KEY }).base(
   process.env.AIRTABLE_BASE
 );
 
 exports.findVolunteerByEmail = async email => {
-  const record = await base("Volunteers")
+  const record = await base('Volunteers')
     .select({
       filterByFormula: `({volunteer_email} = '${email}')`
     })
     .firstPage();
   return record ? record[0] : null;
 };
 
+class Request {
+
+  /**
+   * The mapping between a model object an Airtable's fields.
+   * We do this because airtable only supports setting by name,
+   * so we would like a single place to map the name to an internal
+   * symbol: "Email Address" -> "emailAddress"
+   */
+  airtableBindings = {
```

Comment:
On one hand it didn't feel good duplicating the hardcoded airtable field strings all over the place. On the other hand I feel like there needs to be... a better solution. Ideas?

Reply:
yea, i feel like hardcoded strings might be our best bet. we need some normalization

Reply:
Ok I'll get rid of this complexity and just hardcode some strings. When the app starts dealing with Airtable more comprehensively we can rethink it.